run-tests.ps1

```diff
@@ -10,6 +10,8 @@ function RunTest([string] $project, [string] $description,[bool] $skipBuild = $f
        $cmdargs += "--filter", "$filter"
     }
 
+    $cmdargs += "--blame"
```

Comment:
I added this temporarily to figure out which test was crashing. But thinking about, I wonder if I should just leave it here. Seems like a good thing to have?